tests/integration/targets/idrac_secure_boot/tests/export_all_certs.yaml

```diff
@@ -0,0 +1,68 @@
+# Dell OpenManage Ansible modules
+# Copyright (C) 2024 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+---
+- name: Test that we have an iDRAC host, iDRAC username and iDRAC password
+  ansible.builtin.fail:
+    msg: 'Please define the following variables: idrac_ip, idrac_user and
+     idrac_password.'
+  when: 'idrac_ip is not defined or idrac_user is not defined or idrac_password
+   is not defined'
+
+- block:
+    - ansible.builtin.include_tasks: _helper.yml
+
+    - ansible.builtin.include_tasks: _configure_generic_bios_attributes.yml
+
+    - name: Export all policy certificate
+       (check mode - changes expected)
+      dellemc.openmanage.idrac_secure_boot: &export_cert
+        export_certificates: true
+        database:
+          - "{{ idrac_secure_boot_dir_path_1 }}"
+        disallow_database: "{{ idrac_secure_boot_dir_path_2 }}"
+        KEK:
+          - "{{ idrac_secure_boot_dir_path_3 }}"
+        platform_key: "{{ idrac_secure_boot_dir_path_4 }}"
+        restart: true
+        restart_type: GracefulRestart
+        job_wait: true
+        job_wait_timeout: 1000
+      register: cm_changes
+      check_mode: true
+
+    - name: Verify - Export all policy certificate
+       (check mode - changes expected)
+      ansible.builtin.assert:
+        that:
+          - cm_changes.changed
+          - cm_changes.msg == 'Changes found to be applied.'
+
+    - name: Get time details
+      ansible.builtin.include_tasks: _get_time_details.yml
+
+    - name: Export all policy certificate (normal mode)
+      dellemc.openmanage.idrac_secure_boot:
+        <<: *export_cert
+      register: export_cert
+
+    - name: Verify - Export all policy certificate (normal mode)
+      ansible.builtin.assert:
+        that:
+          - not export_cert.changed
+          - export_cert.msg == "Successfully exported the SecureBoot
+           certificate."
+
+    # - name: Verify - Secure boot Log entries
```

Comment:
Remove the commented code if not needed, please check other files too.

Reply:
done